src/AppBanner/styles.ts

```diff
@@ -1,163 +1,13 @@
-
-        html {
-            box-sizing: border-box;
-        }
-
-        *,
-        *:before,
-        *:after {
-            box-sizing: inherit;
-        }
-    `,
-
-    contentContainer: css`
-        align-items: stretch;
-        display: flex;
-        flex-direction: column;
-        position: relative;
-        margin: 0 auto;
-        width: 100%;
-        max-width: 740px;
-
-        ${from.desktop} {
-            max-width: 980px;
-            flex-direction: row;
-            min-height: ${imgHeight}px;
-        }
-
-        ${from.leftCol} {
-            max-width: 1140px;
-        }
-
-        ${from.wide} {
-            max-width: 1300px;
-        }
-    `,
-
-    topLeftComponent: css`
-        width: 93%;
-        padding: ${space[4]}px;
-        min-height: ${imgHeight}px;
-        ${from.desktop} {
-            width: 60%;
-        }
-    `,
-
-    bottomRightComponent: css`
-        display: flex;
-        justify-content: center;
-        width: 100%;
-        max-height: 100%;
-
-        ${from.desktop} {
-            align-self: flex-end;
-            max-width: 40%;
-            padding-right: ${space[4]}px;
-            max-width: 42%;
-            justify-content: flex-end;
-        }
-
-        ${from.leftCol} {
-            padding-right: 0;
-            justify-content: space-between;
-        }
-
-        ${from.wide} {
-            max-width: 45%;
-        }
-    `,
-
-    heading: css`
-        ${headline.small({ fontWeight: 'bold' })};
-        margin: 0;
-        max-width: 100%;
-
-        ${from.mobileLandscape} {
-            ${headline.small({ fontWeight: 'bold' })};
-        }
-
-        ${from.tablet} {
-            max-width: 100%;
-        }
-    `,
-
-    paragraph: css`
-        ${body.medium()}
-        line-height: 135%;
-        margin: ${space[5]}px 0 ${space[5]}px;
-        max-width: 100%;
-        color: ${bodyColor};
-
-        ${from.phablet} {
-            max-width: 90%;
-        }
-
-        ${from.tablet} {
-            max-width: 100%;
-        }
-
-        ${from.desktop} {
-            font-size: 20px;
-            margin: ${space[3]}px 0 ${space[4]}px;
-            max-width: 42rem;
-        }
-
-        ${from.leftCol} {
-            max-width: 37rem;
-        }
-
-        ${from.wide} {
-            max-width: 42rem;
-        }
-    `,
-
-    cta: css`
-        font-weight: 700;
-        margin-top: ${space[5]}px;
-        margin-right: ${space[3]}px;
-        display: inline-block;
-        color: ${palette.neutral[20]};
-    `,
-
-    smallRightSpacer: css`
-        margin-right: ${space[3]}px;
-    `,
-
-    primaryButton: css`
-        margin-right: ${space[3]}px;
-        background-color: ${palette.neutral[20]};
-        color: ${palette.neutral[100]};
-
-        &:hover {
-            background-color: #111;
-        }
-    `,
-
     secondaryButton: css`
         color: ${palette.neutral[20]};
     `,
 
-    packShot: css`
+    image: css`
```

Comment:
I'm interested in why this isn't also a common style? (But no harm, I know we're deliberately duplicating stuff, which is fine)

Reply:
Hey good question. Mostly because we hadn't received the real image when we did this work and I was expecting that the different shape (from the mockup) and transparency would mean the styles would have to be altered slightly for the Guardian in 2020 banner.